sql/core/src/main/scala/org/apache/spark/sql/jdbc/PostgresDialect.scala

```diff
@@ -307,24 +307,23 @@ private object PostgresDialect extends JdbcDialect with SQLConfHelper {
   override def convertJavaTimestampToTimestamp(t: Timestamp): Timestamp = {
     // Variable names come from PostgreSQL "constant field docs":
     // https://jdbc.postgresql.org/documentation/publicapi/index.html?constant-values.html
-    val POSTGRESQL_DATE_NEGATIVE_INFINITY = -9223372036832400000L
-    val POSTGRESQL_DATE_NEGATIVE_SMALLER_INFINITY = -185543533774800000L
```

Comment:
`*_SMALLER_INFINITY`s are for PgResultSet to identify date infinities internally, they are supplanted with the larger abs-values to clients